src/components/settings/subjects/SubjectsPanel.tsx

```diff
@@ -10,12 +10,12 @@ export const SubjectsPanel = () => {
   const { query } = useSubjects();
 
   return (
-    <Flex direction={'column'} flexGrow={1} overflowY={'hidden'} p={10} m={-10}>
+    <>
       <SubjectCreateModal isOpen={isCreateModalOpen} onClose={onCreateModalClose} />
       <Flex
         direction={'column'}
         borderRadius={10}
-        backgroundColor={'gray.700'}
+        backgroundColor={'gray.750'}
```

Comment:
If colors are used in multiple places (or they can be used potentially in the future) maybe it's better to put them together in a single file and create constants?

Reply:
This way 1. changes would be done in a single place (not each file separately) and 2. one wouldn't have to think much about which color to use

Reply:
Nvm, that's done already.